Gemfile

```diff
@@ -80,7 +80,6 @@ gem 'rsolr', '>= 1.0', '< 3'
 gem 'bootstrap', '~> 4.0'
 gem 'popper_js'
 gem 'twitter-typeahead-rails', '0.11.1.pre.corejavascript'
-gem 'jquery-rails'
```

Comment:
Adding a Dockerfile should definitely not remove this gem.

Reply:
This is redundant it looks like. It was listed twice in the file which then bundler complained about.

Reply:
@botimer Thoughts?

Reply:
It is weird that it is listed twice and I never got the error! Thanks for removing it.